### ISSUE TITLE
Research ways to get user image feedback

### ISSUE THREAD

**Issue by wingman-jr-addon** [wingman-jr-addon/wingman_jr#40]
-While maintaining privacy
-With easy-to-use, free tech
-And not enabling psychological dark patterns (e.g. now I'll submit all the pron)

**Comment by wingman-jr-addon** [comment]
Asking on Mozilla's Discourse [here](https://discourse.mozilla.org/t/user-submitted-data-and-privacy/52717/2)

**Comment by wingman-jr-addon** [comment]
The best thought I have so far is doing something time-based. Rather than having users click on good or bad images - as that can encourage looking for the bad - simply have a one-button press that submits recent URL's and then it can be sorted out on the back end.

**Comment by wingman-jr-addon** [comment]
As a start, I now at least have a text-based feedback link in the extension.

**Comment by wingman-jr-addon** [comment]
As an update, I have gotten zero text-based feedback to date (other than exit surveys, which I have gotten a few of). As a consequence, I'm only going to label this as a low priority item because lack of feedback may indicate that this would not be successful.

**Comment by wingman-jr-addon** [comment]
An idea around technique - use Google Forms:
https://trevorfox.com/2015/06/dynamically-pre-fill-google-forms-with-mailchimp-merge-tags/

**Comment by AMTitan** [comment]
You could try having a system where if an image is blocked you can right click on it to show the original. Once it is the original you can right click on the image again and report it as a false positive. If the image was never blocked you can right click and say an image should be blocked.

I don't know if this will do as you said before with "encourage looking for the bad", but there are some sites that I am pretty sure they don't have an image that should be blocked. This would give me a good reason to check to see the unblocked version of the image.

**Comment by wingman-jr-addon** [comment]
@arthurmelton thanks for the feedback on this long-open issue!
Yes, I think one of the challenges of any of the more "individual" image reporting options is that I think some people may start to use the plugin primarily for reporting images. Some will just check and report every once in a while, but particularly for those users who may be struggling with an addiction I think it may serve to "gamify" looking at images they are trying to avoid. So there's the psychological aspect here that is a bit tricky.
The other aspect is the technological one. I'm not keen to set up a dedicated reporting site as I'd rather direct my limited energies towards working on the addon itself. Ideas are definitely welcome there too.
